backends-common/elasticsearch-v8/src/main/java/org/apache/james/backends/es/v8/ElasticSearchIndexer.java

```diff
@@ -113,7 +113,7 @@ public Mono<BulkResponse> delete(List<DocumentId> ids, RoutingKey routingKey) {
         return client.bulk(bulkBuilder.build())
             .onErrorResume(exception -> {
                 LOGGER.warn("Error while deleting index", exception);
-                return Mono.empty();
+                return Mono.error(exception);
             });
```

Comment:
Log then rethrow is an anti-pattern...

Here we likely should not do anything I guess.

Reply:
There is tests checking that an exception is propagated correctly to the top... if I leave Mono.empty() well the test just hangs forever... which sounds logical as we don't propagate the exception... I don't get how that was passing before though?

Reply:
alright get rid of the onErrorResume and propagate the exception  to the caller, got it

Reply:
I mean:
 - Remove the log as we are o longer managing the exception here
 - Then the onErrorResume blocks becomes more or less useless and can be removed.